public/app/ml-worker-fetch-cache.js

```
@@ -452,14 +460,30 @@ function _ensureProgressPanel(keys) {
   panel.appendChild(title);
 
   for (const key of keys) {
-    const row   = document.createElement('div');
+    const row = document.createElement('div');
```

ESLint

[error] 'document' is not defined.

(no-undef)